CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+
+- Run a `containerdebug` process in the background of each Airflow container to collect debugging information ([#557]).
```

Comment:
We only run it in one container per Pod, right? Given that the paths are all the same, they would trample all over each other otherwise.

Reply:
Yes, I meant the container named "airflow"